globus_cli/commands/whoami.py

```diff
@@ -32,9 +31,8 @@ def whoami_command(linked_identities):
     try:
         res = client.oauth2_userinfo()
     except AuthAPIError:
-        safeprint(
-            "Unable to get user information. Please try " "logging in again.",
-            write_to_stderr=True,
+        click.echo(
+            "Unable to get user information. Please try " "logging in again.", err=True
```

Comment:
This is weird. Probably happened when we did the `black` transition.

Reply:
Almost for sure. I've caught some of these in other projects too.